app/Services/ValidationService/ValidationRules.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Services;
+
+class ValidationRules
+{
+    const RULES = [
+        'post' => [
+            'author_id' => [
+                'required' => true,
+                'integer'  => true,
+                'exists'   => 'users,id'
+            ],
+            'slug'      => [
+                'required' => true,
+                'string'   => true,
+                'unique'   => 'posts',
+                'regex'    => '/^[a-zA-Z0-9_-]*$/'
+            ],
+            'title'     => [
+                'required' => true,
+                'string'   => true,
+            ],
+            'excerpt'   => [
+                'max' => 255,
+            ],
+            'status_id' => [
+                'required' => true,
+                'integer'  => true,
+//                'exists'   => 'post_statuses.id',
+            ],
+        ],
+    ];
+
+    public static function get(string $entityName): array
+    {
+        $resultRules = self::concatenateRules(self::RULES[$entityName]);
```

Comment:
Use `Arr::get()` to access array as you cannot predict what argument can be provided as `$entityName`. So there is possibility to get "Undefined index" notice.
It's a good standard to use `Arr::get()` everywhere in the code (unless you are sure that it will not fire error notices). So please wrap all array usages with it.